app/(main)/login/page.tsx

```diff
@@ -82,10 +82,19 @@ const Login = (): React.JSX.Element => {
    * Handles the form submission.
    * @param {LoginUserSchemaType} data - The data from the form.
    */
-  const onSubmit: SubmitHandler<LoginUserSchemaType> = async (data) => {
-    setIsLoading(true);
-    await login(data);
-    setIsLoading(false);
+  const onSubmit: SubmitHandler<LoginUserSchemaType> = async (
+    data: LoginUserSchemaType,
+  ): Promise<void> => {
+    try {
+      setIsLoading(true);
+      await login(data);
+      setIsLoading(false);
```

Comment:
I think you can remove the `setIsLoading(false)` right after the `await login(data)`.  The  `setLoading(false)` in the finally section should set this to false as long as the error isn't triggered.

Reply:
Thank you for explaining this in our chat the other night. I've just made that change in the latest commit.

Reply:
@chris-nowicki if this is resolved, please resolve the conversation. Ty!